src/pages/en/guides/content-collections.md

```diff
@@ -241,21 +257,21 @@ const { post } = Astro.props;
 
 ## Rendering page content from entries
 
-To render the content of your Markdown and MDX entries, use the `<Content />` component returned by the `renderEntry()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
+To render the content of your Markdown and MDX entries, use the `<Content />` component returned by the `render()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
 
-### `renderEntry()`
+### `render()`
 
-`renderEntry()` is a function that takes an entry retrieved by `getEntry()` as a parameter and returns a `<Content />` component containing the rendered content of the Markdown or MDX file. 
+`render()` is an extension function every content collection entry that returns a `<Content />` component. This contains the rendered content of the Markdown or MDX file. 
```

Comment:
Think this got a bit mangled in the edit. Here’s my suggested fix:

```suggestion
Every content collection entry includes a `render()` function that returns a `<Content />` component. This component will render the content of the Markdown or MDX file. 
```

Reply:
This looks good to me Chris. 

*If* we rework the paragraph above to more generically describe `render()` and each of these level 3 headings are the three things you can access with it, then this could be very slightly tweaked to sound less like it's an introduction of the `render()` function itself, and hit `<Content />` as one of the things you can use the aforementioned `render()` function to access. But, non-blocking either way! Could totally merge like this.